scripts/artifacthub/hub.go

```diff
@@ -221,11 +224,38 @@ func addArtifactHubMetadata(sourceDirectory, destinationPath, ahBasePath string,
 	}
 }
 
-func checkVersion(artifactHubMetadata *ArtifactHubMetadata, constraintTemplate map[string]interface{}, newConstraintTemplateHash string) error {
-	// compare hash
-	if artifactHubMetadata.Digest != newConstraintTemplateHash {
+func checkVersion(artifactHubMetadata *ArtifactHubMetadata, constraintTemplate map[string]interface{}, githubSourceRelativePath string) error {
+	// compare hash with template.yaml in github
+	githubTemplateURL := sourceURL + githubSourceRelativePath
+	resp, err := http.Get(githubTemplateURL)
+	if err != nil {
+		fmt.Println("error while getting constraint template from github")
+		panic(err)
+	}
+	if resp.StatusCode == http.StatusNotFound {
+		fmt.Printf("constraint template %s not found in github. It is likely that constraint template is being updated locally and not merged to github yet.\n", githubSourceRelativePath)
+		
+		return nil
```

Comment:
This test probably needs to be updated? https://github.com/open-policy-agent/gatekeeper-library/blob/master/scripts/artifacthub/hub_test.go#LL184C24-L184C24

Reply:
I updated the entire test method.